docker-compose.yml

```diff
@@ -431,6 +431,8 @@ services:
         target: /etc/nginx/certs/${DOMAIN:?error}.crt
       - source: nginx_tls_key
         target: /etc/nginx/certs/${DOMAIN:?error}.key
+      - source: nginx_stream
+        target: /etc/nginx/toplevel.conf.d/mqtt.conf
```

Comment:
How will nginx know about this custom `/etc/nginx/toplevel.conf.d` configuration directory?

Reply:
https://github.com/nginx-proxy/nginx-proxy/tree/main/docs#tcp-and-udp-stream